presto-main/src/main/java/com/facebook/presto/type/BooleanOperators.java

```diff
@@ -100,6 +101,13 @@ public static double castToDouble(@SqlType(StandardTypes.BOOLEAN) boolean value)
     }
 
     @ScalarOperator(CAST)
+    @SqlType(StandardTypes.REAL)
+    public static long castToReal(@SqlType(StandardTypes.BOOLEAN) boolean value)
+    {
+        return value ? floatToRawIntBits(1) : floatToRawIntBits(0);
```

Comment:
I personally prefer to use `1.0f` and `0.0f` here. @haozhun what's your opinion? (We are using `1` and `0` in `castToDouble`, though.